src/utils/daemon.rs

```diff
@@ -110,7 +111,9 @@ pub fn start_daemon(
     // at least start web server
     info!("Starting web server");
 
-    crate::Server::start(None, false, db, config, build_queue)?;
+    let storage = Arc::new(Storage::new(db.clone()));
+
+    crate::Server::start(None, false, db, config, build_queue, storage)?;
```

Comment:
The number of parameters keeps going up! 😆 It's fine for now, but we might want to think about factoring out some of these into a struct.

Reply:
Yeah, we should eventually create a `trait Context`.